projects/04-burger-queen-api/e2e/globalSetup.js

```diff
@@ -61,13 +61,13 @@ const createTestUser = () => fetchAsAdmin('/users', {
 })
   .then((resp) => {
     if (resp.status !== 200) {
-      throw new Error('Could not create test user');
+      throw new Error(`Response ${resp.status} - Could not create test user`);
```

Comment:
Es sólo un detalle, quizás da lo mismo, pero asumiendo que es un mensaje de error, pondría Error en vez de Response:
```suggestion
      throw new Error(`Error ${resp.status} - Could not create test user`);
```

Reply:
Hmmm es el response del server con el status, por eso puse "Response" dejame revisar quiza uso los dos.